<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestTapis</class>
 <widget class="QWidget" name="TestTapis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestTapis</string>
  </property>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>191</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tapis A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetTapisA"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSortisTapisA">
      <property name="text">
       <string>Sortir du tapis A </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonRazTapisA">
        <property name="text">
         <string>RAZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberA"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>191</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tapis B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetTapisB"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSortirTapisB">
      <property name="text">
       <string>Sortir du tapis B</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonRazTapisB">
        <property name="text">
         <string>RAZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>191</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tapis C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetTapisC"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSortirTapisC">
      <property name="text">
       <string>Sortir du tapis C</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonRazTapisC">
        <property name="text">
         <string>RAZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberC"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>151</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rouleau à sortir: </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Référence: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditReference"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Diamètre :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDiametre"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSortirRouleau">
        <property name="text">
         <string>Sortir le rouleau</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
